webpages/settings/components/addon-body.html

```diff
@@ -141,17 +149,24 @@
     font-size: 13px;
   }
   .addon-credits a,
+  .related-addons a,
   .addon-license a {
     color: var(--blue-text);
+    cursor: pointer;
   }
 
-  .addon-credits > span {
+  .addon-credits > span,
+  .related-addons span {
     margin-inline-end: 0.5rem;
   }
 
-  .addon-credits > span:not(:first-child):not(:last-child)::after {
+  .addon-credits > span:not(:first-child):not(:last-child)::after,
+  .related-addons span:not(:first-child):not(:last-child)::after {
     content: ",";
   }
+  .related-addons span:first-child {
+    font-weight: bold;
+  }
```

Comment:
These CSS changes only apply some .addon-credits styles to .related-addons as well. There are no new rules.